plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -125,6 +130,12 @@ func (c *nodePlugin) Admit(a admission.Attributes) error {
 			return admission.NewForbidden(a, fmt.Errorf("may only update PVC status"))
 		}
 
+	case svcacctResource:
+		if c.features.Enabled(features.TokenRequest) {
+			return c.admitServiceAccount(nodeName, a)
+		}
+		fallthrough
```

Comment:
not sure if matters if we deny above if the feature is disabled, but prefer `return nil` over `fallthrough`

Reply:
done.